### ISSUE TITLE
Banlist page

### ISSUE THREAD

**Issue by VjGamingUK** [TotalFreedom/TotalFreedomMod#366]
Why? See here: http://totalfreedom.boards.net/thread/17126/ops-away-bans

OPs getting away with bans, therefore we need to see the banlist, but it's in the files, therefore why not have a banlist page? on http://madgeekonline.com/apps/banlist
and their name would appear like this
IPs for those who were range banned

Example template what it looks like:
[Username goes here]:
[Ban Reason]
## [IPs]

So the suggestion is for admins to see the banlist to find out why they were banned or not, as players get away with unbans for lying on their appeals saying "no reason" or "banned offline" or "I don't know why I was banned" or etc...

Whenever someone gets banned, it'll auto-add their name to the banlist, so it's automatic just like the permban list and the logviewer, when someone gets unbanned then their name will be removed.

**Comment by RobinGall2910** [comment]
@SmokeSnake Where is it gonna be at if it wasn't the HTTPD? There's nothing else to stuff it in xD

**Comment by ItzLevvie** [comment]
I could add this, i have it on my desktop.

**Comment by jjkatz613** [comment]
Sorry for Bumping but this is a good idea that never got taken care of

**Comment by Wild1145** [comment]
I agree, but I would suggest some changes to the implementation of the idea, If someone tags me again in a day or two I will write up my ideas properly on a potentially better way to implement this idea.
